virtualize.sh

```diff
@@ -16,80 +16,49 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-set -e
-set -x
+
+### Intial definitions
+
+set -ue
 
 ORIG=$(cd $(dirname $0); pwd)
 PREFIX=$USER
 installserver_name="os-ci-test4"
+router_name="router"
+routerip=""
 installserverip=""
+virthost="localhost"
+
+### Handler Functions
+
 usage () {
-      set +x
-      echo "
-usage: virtualize.sh [OPTION] workdir1 workdir2 etc
+        echo "
+usage: $0 [OPTION]
 Collect architecture information from the edeploy directory as generated
-by config-tools/download.sh.
+by config-tools/download.sh and use the virtulizor.py to boostrap a SpS platform.
 
-optinal arguments:
-  -H|--hypervisor=name: change the hypervisor name, default (${hypervisor})
 arguments:
-virtualize.sh will use the first argument as the location of the
-SpinalStack environment to deploy. It will then upgrade the newly deployed
-SpinalStack to the following environment directory.
-
-For example: ./virtualize.sh I.1.2.1 I.1.3.0 I.1.3.1
-will deploy environment I.1.2.1 and upgrade to I.1.3.0 and then I.1.3.1."
-      exit 1
+    -H|--hypervisor=name          Set the hypervisor hostname, default (${virthost})
+    -d|--debug                    Set the debug mode for this script, default: disabled
+    -w|--wordkir=dir1,dir2,...    Workdir List, default: None
+    -v|--virt=virt_platform.yml   Set the path to the infra's yaml
```

Comment:
i don't see the benefit of this new parameter. So far, the virt_playtform.yml was in the workdir. I don't understand why you need to change that.